test/Bridges/utilities.jl

```diff
@@ -4,10 +4,25 @@ function test_num_constraints(bridged_mock, F, S, n)
     @test ((F, S) in MOI.get(bridged_mock, MOI.ListOfConstraints())) == !iszero(n)
 end
 
+function warn_incomplete_list_num_constraints(BT, list_num_constraints)
+    for (S,) in MOIB.added_constrained_variable_types(BT)
+        F = MOIU.variable_function_type(S)
+        if !any(c -> c[1] == F && c[2] == S, list_num_constraints)
+            @warn("Bridges of type $BT add constrained variable in $S but their number is not tested.")
```

Comment:
Warn or make this an error?

Reply:
This is used only by us in tests, it's useful to have it as a warning so that we can see the rest of the test. The advantage of an error though is that it will fail in CI so an error might be better.

Reply:
> The advantage of an error though is that it will fail in CI so an error might be better.

This is what I was thinking.

Reply:
It was indeed the right choice, it just had its first use case:
https://travis-ci.org/JuliaOpt/MathOptInterface.jl/builds/580800484